source/math/FloatBox2D.ooc

```diff
@@ -80,9 +87,19 @@ FloatBox2D: cover {
 		This new(left, top, width, height)
 	}
 	contains: func (point: FloatPoint2D) -> Bool {
-		this left <= point x && point x < this right && this top <= point y && point y < this bottom
+		this left <= point x && point x <= this right && this top <= point y && point y <= this bottom
 	}
 	contains: func ~box (box: FloatBox2D) -> Bool { this intersection(box) == box }
+	contains: func ~FloatPoint2DVectorList (list: FloatPoint2DVectorList) -> VectorList<Bool> {
+		result := VectorList<Bool> new()
+		for (i in 0..list count) {
+			if (this contains(list[i]))
+				result add(true)
+			else
+				result add(false)
+		}
+		result
+	}
```

Comment:
Instead of a list of `Bool` for each index in the list, wouldn't it be better to have list of `Int` with "valid" indices?